pkg/filler/vector.go

```diff
@@ -5,27 +5,19 @@ import (
 	"github.com/anubis-game/apiserver/pkg/vector"
 )
 
-type Vector struct {
-	Qdr int
-	Agl int
-	Obx int
-	Oby int
+func (f *Filler) Vector() *vector.Vector {
+	return <-f.vec
 }
 
-func (f *Filler) Vector(uid byte) *vector.Vector {
-	var vfl Vector
-	{
-		vfl = <-f.vec
-	}
-
```

Comment:
I think we really don't want the byte ID to be part of the Vector. Removing it has a huge rat tail of all sorts of consequences, which in itself tells me that the current design is extremely flawed.

Reply:
Resolved in #52.